File: AoC-2015/Day02/Problem1.py
ans = 0

with open("input.txt", "r") as f:
    for line in f:
        a, b, c = map(int, line.strip().split('x'))    
        ans += 2*((a*b) + (b*c) + (c*a)) + min(a*b, b*c, c*a)

print(ans)

File: AoC-2025/Day01/Problem1.java
import java.io.FileReader;
import java.io.IOException;
import java.io.BufferedReader;

public class Problem1
{
    public static void main(String[] args)
    {
        try(BufferedReader br = new BufferedReader(new FileReader("Day01/input.txt")))
        {
            String line;
            int ans = 0, pos = 50;

            while((line = br.readLine()) != null)
            {
                char dir = line.charAt(0);
                int steps = Integer.parseInt(line.substring(1));

                if(dir == 'L') pos = (pos - steps + 100)%100;
                else pos = (pos + steps)%100;

                if(pos == 0) ans++;
            }

            System.out.println(ans);
        }
        catch (IOException e)
        {
            e.printStackTrace();
        }
    }
}